build/steps-build-ios.yml

```diff
@@ -88,7 +88,7 @@ steps:
   inputs:
     target: 'ios'
     buildName: '$(MajorMinorPatch)'
-    buildNumber: '$(PreReleaseNumber)'
+    buildNumber: '$(buildNumber)'
```

Comment:
Same comment as previous.

Reply:
The logic that supported this variable has now been removed.